src/main/java/core/basesyntax/service/impl/ReadFromFileImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReadFromFile;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class ReadFromFileImpl implements ReadFromFile {
+
```

Comment:
redundant empty line

Reply:
Maven clean package thoughts that its not redundant:
[ERROR] /home/lubus/IdeaProjects/Mate-academy/JavaCore/SOLID/src/main/java/core/basesyntax/service/impl/ReaderImpl.java:8: 'CLASS_DEF' should be separated from previous statement. [EmptyLineSeparator]